debian/xenial/kubelet/lib/systemd/system/kubelet.service

```diff
@@ -1,10 +1,12 @@
 [Unit]
-Description=Kubernetes Kubelet Server
-Documentation=https://github.com/kubernetes/kubernetes
+Description=kubelet: The Kubernetes Node Agent
+Documentation=http://kubernetes.io/docs/
 
 [Service]
-ExecStart=/var/lib/kubelet/kubelet-wrapper
+ExecStart=/usr/bin/kubelet
```

Comment:
Interesting, we're using EnvironmentFile=-/etc/default/$APP  in our systemd jobs.  I didn't actually know about this drop-in bit, which is very interesting. I like that we can just expose that and users can literally spray configuration all over that directory and it just gets absorbed.

Does this retain the same behavior as the EnvironmentFile minus the requirement for appending to the ExecStart stanza?